hashintel/hash
# H-2300, H-2343, H-2341, H-2334: move `/drafts` to `/actions`  + other related crash fixes/changes

## 🌟 What is the purpose of this PR?




This PR:
- moves the `/drafts` page to `/actions` (H-2300)
- stops the creation of a notification when a draft entity is created in the `inferEntities` flow
- adds the `Actions`/`Notifications` sidebar items, as children of the existing `Inbox` sidebar item (H-2300)
- adds a dedicated "Actions" count/link to the header (H-2300)
- adds tooltips to the header notifications/actions counters/links
- hides the "Bulk Action" dropdown when no drafts are selected (H-2341)
- fixes a crash that previously occurred when a new draft entity was added to the drafts page while it was open, related to the subgraph that was being generated for the draft entity modal (H-2334)
- fixes a crash that previously occurred when a draft entity of a new type was added to the drafts page while it was open, related to how the filter state being generated (H-2334)
- when a draft entity of a new type is added to the drafts page, the type is checked in in the filter state as the entity would otherwise not appear

## 🔗 Related links





- H-2300
- H-2343
- H-2341
- H-2343

## 🔍 What does this change?




See description.

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:


### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:


## ⚠️ Known issues



## 🐾 Next steps



## 🛡 What tests cover this?



Manual testing.

## ❓ How to test this?



1. Go to the `/actions` page, and observe the refactored sidebar/header
2. Via the browser extension create a draft of a new type whilst the `/actions` page is open
3. Ensure that there is no crash, and the entity is immediately visible (i.e. its type is added and not filtered out in the filter sidebar)

## 📹 Demo



<img width="1510" alt="image" src="https://github.com/hashintel/hash/assets/42802102/da282d7e-257a-4c0a-aa58-9e35150a9d13">
